### ISSUE TITLE
The login error is not displayed to user

### ISSUE THREAD

**Issue by thecodeholic** [thecodeholic/laravel8-vue3-survey#1]
![chrome_pOwYFRLJPu](https://user-images.githubusercontent.com/4627922/148632617-4580a438-89c7-47db-9988-ea6882d8efee.png)

**Comment by Azam2926** [comment]
this might be solution (in my mind) https://github.com/thecodeholic/laravel-vue-survey/pull/4

**Comment by Dingo497** [comment]
This project is mine. Everything. They just edited it a bit. But I'm glad if my code helped someone :)

**Comment by horlas08** [comment]
> This project is mine. Everything. They just edited it a bit. But I'm glad if my code helped someone :)

Yes have go through your GitHub I saw the project is yours but u can't claim it yours because there is a new things added

**Comment by TylerByte666** [comment]
@Dingo497 Claiming something like that on an open source platform is very sad. Next time mark your repo as private, then we won't have to listen to this BS.

**Comment by horlas08** [comment]
> @Dingo497 Claiming something like that on an open source platform is very sad. Next time mark your repo as private, then we won't have to listen to this BS. You were born in '97 making you still very naive.



> @Dingo497 Claiming something like that on an open source platform is very sad. Next time mark your repo as private, then we won't have to listen to this BS. You were born in '97 making you still very naive.

As u can see he didn't have any issue with copy or no copy he said at the end that he is glad that his code help numerous

**Comment by TylerByte666** [comment]
As I can see, you struggle with english. He does have a problem with it or why say something at all?
The way I see it, you have a problem too. As you accept his claim with no evidence. Birds of a feather... You assume I can't read!

**Comment by horlas08** [comment]
> As I can see, you struggle with english. He does have a problem with it or why say something at all? The way I see it, you have a problem too. As you accept his claim with no evidence. Birds of a feather... You assume I can't read! We have zero evidence to prove anything he said, yet you blindly defend a butt-hurt person whose entire repositories are not in english! Go code! Give me a break! BS!

Go through his channel before u start pooping rubbish from mouth as u can see In my comment I went through his repo before I make I comment not a dump like you replying over 2month+ comment which is irrelevant your reaction already tell public when u start learning Laravel I don't have your time ok

**Comment by TylerByte666** [comment]
So i guess your Pro-nouns are: "idiot|cantspeekenrish|I cannot code|" Why you hiding your repo you p*ssy.

**Comment by horlas08** [comment]
That

> So i guess your Pro-nouns are: "idiot|cantspeekenrish|I cannot code|" Why you hiding your repo you p*ssy. I just told your illiterate ass that Iwent through his repo. Fix your english and head you little kid!

Yes that's why u use been instead of being on your repo 
Kindly rephrase it
Bush MVVM & Bloc architect from Africa. Being a programmer since 2010, but hacker since birth. Let me tell you monkey jokes...

**Comment by horlas08** [comment]
😂😂😂 sorry I can feel your pain

**Comment by horlas08** [comment]
100% my foot

> pussy, check this monkey! You moron it's 100% correct. Why don't you go watch CP?

💯 my foot 😂😂

**Comment by TylerByte666** [comment]
Go touch an underage kid, before I TEACH YOU What ethical hacking means

**Comment by horlas08** [comment]
Lolz with these ur flutter and JavaScript level

**Comment by TylerByte666** [comment]
"I have been fucking @horlas08 in the ass since 2010" Happy, cunt!

**Comment by horlas08** [comment]
@TylerByte666 I guess your generation is a gay, sorry did you even have one

**Comment by TylerByte666** [comment]
ASk your mom afte she chockes on my dick, youre a new monkey joke :)

**Comment by horlas08** [comment]
> ASk your mom afte she chockes on my dick, youre a new monkey joke :) It's 2023, you can't be homophobic you homolicker

Next time kindly rephrases your sentence before dropping on a public, ok I don't have your time check the correct and sleep bye 
Correction:-
Ask your mom after she chokes on my dick, you're a new monkey joke :) It's 2023, you can't be homophobic you homolicker

**Comment by horlas08** [comment]
> ASk your mom afte she chockes on my dick, youre a new monkey joke :) It's 2023, you can't be homophobic you homolicker

And also I hope by now you should have corrected your repo

**Comment by horlas08** [comment]
> ASk your mom afte she chockes on my dick, youre a new monkey joke :) It's 2023, you can't be homophobic you homolicker

Have a nice day

**Comment by TylerByte666** [comment]
I fucked Martin's mama

**Comment by TylerByte666** [comment]
I know your real name ans surname now, only a matter of time...
